testing/tiptop/APKBUILD

```diff
@@ -0,0 +1,28 @@
+#-*-mode: Shell-script; coding: utf-8;-*-
+# Maintainer: Mitch Tishmack <mitch.tishmack@gmail.com>
+pkgname=tiptop
+pkgver=2.3
+pkgrel=0
+pkgdesc="Tiptop: Hardware Performance Counters for the Masses"
+url="http://tiptop.gforge.inria.fr"
+arch="x86_64"
+license="gpl2"
+depends=""
+makedepends="byacc flex linux-headers ncurses-dev libxml2-dev"
+subpackages="${pkgname}-doc"
+source="
+	http://${pkgname}.gforge.inria.fr/releases/${pkgname}-${pkgver}.tar.gz
+"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	./configure --prefix=/usr
+	make
+}
+
```

Comment:
Could you add a smoke-test `check()` that simply runs the resulting binary with e.g. `--version`?